stats/aggregation_value.go

```diff
@@ -81,9 +81,9 @@ func (a *AggregationCountValue) String() string {
 	return fmt.Sprintf("{%v}", *a)
 }
 
-// AggregationDistributionValue is the aggregated data for an
+// TestingAggregationDistributionValue is the aggregated data for an
 // AggregationDistributionFloat64  or AggregationDistributionInt64.
-type AggregationDistributionValue struct {
+type TestingAggregationDistributionValue struct {
```

Comment:
Why do you change this to be Testing. Seems that this is used in the api not only for testing.

Reply:
Yes @bogdandrutu . These are the actual values that need to be exported. They are not for testing.